_maps/map_files/KiloStation/KiloStation.dmm

```diff
@@ -12710,7 +12710,7 @@
 	height = 22;
 	id = "whiteship_home";
 	name = "SS13: Auxiliary Dock, Station-Port";
-	roundstart_template = /datum/map_template/shuttle/mining_common/kilo;
+	roundstart_template = null;
```

Comment:
isnt this already gotten from the parent? this will just make a duplicate unnecessary var

Reply:
i dont know, literally, i clicked the big circle button that says "revert var to default" and thats what it was, null is also the default var as i confirmed checking other docks, i did this within sdmm not through file editing, so this *shouldnt* be a duplicate var, i assume sdmm is smart enough to not do that

Reply:
but if you think just deleting the var (which is all i actually did, the new line was what sdmm did), then ill do that, im new to byond code

Reply:
u should check in sdmm options to remove all duplicate vars and then save the map, see if that clears it up, if it doesnt then its ok